notebooks/daniel_notebook.ipynb

```diff
@@ -1,191 +1,26 @@
 {
```

Comment:
<p>From these equations, you start by determining vector directions with unit vectors. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>